packages/elements/src/collapse/index.ts

```diff
@@ -195,15 +202,17 @@ export class Collapse extends BasicElement {
    */
   protected render (): TemplateResult {
     return html`
-      <ef-header part="header" level="${this.level}" @tap="${this.handleTap}">
-        <ef-icon icon="right" slot="left" part="toggle"></ef-icon>
+      <ef-header part="header" level="${this.level}" @tap=${this.handleTap} role="heading" aria-level=${this.ariaLevel}>
+        <div part="header-toggle" id="header-toggle" role="button" tabindex="0" aria-expanded="${this.expanded}" aria-controls="content">
+          ${this.header}
+        </div>
+        <ef-icon icon="right" slot="left" part="toggle" aria-hidden="true"></ef-icon>
         <slot slot="left" name="header-left"></slot>
         <slot slot="right" name="header-right"></slot>
-        ${this.header}
       </ef-header>
-      <div part="content">
+      <div id="content" part="content" role="region" aria-labelledby="header-toggle">
         <ef-panel ?spacing="${this.spacing}" transparent>
-          <slot></slot>
+          <slot ?hidden=${!this.expanded}></slot>
```

Comment:
I do not like setting `hidden` on slot itself, it might not work correctly in IE11, and slot is kind of a special element. I can see two solutions:
1) Make slot optional, e.g. `${this.expanded ? html`<slot></slot>` : undefined}`
2) Set hidden on the panel

Reply:
That is correct, it does not work on IE11.  I will go with solution 1 (Make slot optional) because solution 2 I was tested does not work on Chromium browsers.

Reply:
In solution one it might affect performance because it swaps content in/out DOM tree.

Reply:
Even if it does affect performance (which I doubt, as the slot acts as a placeholder and the actual content is already on the page), the effect can be disregarded. Anything that can be done within a single frame (~24ms) is not noticeable by the user eye.

Reply:
I agree with you.

Reply:
I use a new solution, just add CSS `visibility hidden` to the content when collapsed. it works well.